Equilobe.DailyReport.Models/ReportFrame/IndividualDraftInfo.cs

```diff
@@ -12,7 +12,7 @@ public class IndividualDraftInfo
         public string Username { get; set; }
         public string UniqueUserKey { get; set; }
         public DateTime? LastConfirmationDate { get; set; }
-        public DateTime ReportDate { get; set; }
+        public string ReportDate { get; set; }
```

Comment:
Am pus comment pe clasa model din baza de date. Asta este o clasa de care ma folosesc in tool. O sa pun si aici comment. Si da, fiecare proprietate de aici se salveaza in baza de date in IndividualDraftConfirmations.